applications/GeoMechanicsApplication/license.txt

```diff
@@ -0,0 +1,7 @@
+Kratos - GeoMechanics Application
```

Comment:
@KratosMultiphysics/technical-committee  this is a blocker :-(

our policy is to admit as part of the main repository only BSD-licensed applications, so my understanding is that we cannot have a privative licensed application in the main repository. To Be Discussed in @KratosMultiphysics/technical-committee

Reply:
Hi @RiccardoRossi,

Our ultimate goal is also to use BSD license. However, this needs some investigation to foresee its consequences. We have chosen this license as a temprary solution to allow further developement/use of Geomechanics application. The application is still open-source but limitted to researchers and academia. Therefore, it should be no problem with the users at the moment.

Cheers,

Reply:
@RiccardoRossi : The issue with the license is solved. We switched to BSD license.

Reply:
I have already adjusted the license. I hope the code can be merged to the master